.codeclimate.yml

```diff
@@ -0,0 +1,34 @@
+---
+engines:
+  brakeman:
+    enabled: true
+    exclude_fingerprints:
```

Comment:
how did you configured these? I don't see them mentioned in the docs (https://docs.codeclimate.com/v1.0/docs/brakeman)